zebra-scanner/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "zebra-scanner"
+version = "1.0.0-beta.30"
+authors = ["Zcash Foundation <zebra@zfnd.org>"]
+description = "Blockchain scanner for the Zebra Zcash node"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/ZcashFoundation/zebra"
+edition = "2021"
+
+readme = "../README.md"
+homepage = "https://zfnd.org/zebra/"
+# crates.io is limited to 5 keywords and categories
+keywords = ["zebra", "zcash"]
+# Must be one of <https://crates.io/category_slugs>
+categories = ["cryptography::cryptocurrencies"]
+
+[features]
+
+# Production features that activate extra dependencies, or extra features in dependencies
```

Comment:
Team Decision: Should the scanner be off by default, and enabled by a Rust feature? How will the feature work? (Which crates need the feature, what is it called, etc.)

Should the name be different to the crate name?
For example: `external-wallet` or `wallet-support`

`cargo` automatically enables optional crate dependencies with the same name as features. But how it works is changing in 2024, so we might not want to rely on it.

Reply:
In particular, I'd like to avoid compiling `zcash_client_backend` into production `zebrad` releases until it does something useful. Unused dependencies are a minor security risk, and they also slow down builds.

Reply:
This can wait until we have production APIs.